test/e2e/specs/all-features/navbar/base-config.spec.js

```diff
@@ -103,37 +105,32 @@ describe('Navbar ', function() {
 
     it('should show the "Full Name" of the logged in user in the top right', function () {
         var name = (!process.env.CI ? browser.params.client.full_name : browser.params.client.display_name);
-        expect(element(by.css('login .username-display')).getText()).toBe(name, "user's displayed name is incorrect");
+        expect(element(by.css('.username-display')).getText()).toBe(name, "user's displayed name is incorrect");
     });
 
 
     it('should open the profile card on click of My Profile link', function(done) {
-        var link = element(by.css('login .dropdown-toggle'));
-
-        browser.wait(EC.elementToBeClickable(link), browser.params.defaultTimeout);
-        chaisePage.clickButton(link).then(function() {
-            var profileLink = element(by.id('profile-link'));
-            browser.wait(EC.elementToBeClickable(profileLink), browser.params.defaultTimeout);
-            chaisePage.clickButton(profileLink).then(function() {
-                var modalContent = element(by.css('.modal-content'));
-
-                chaisePage.waitForElement(modalContent);
-                expect(modalContent.isDisplayed()).toBeTruthy();
-                done();
-            }).catch(function (err) {
-                done.fail();
-                console.log(err);
-            });
+        // Get My Profile link element using className username-display
+        var link = element(by.css('.username-display'));
+
+        link.click().then(function() {
+            const profileLink = element(by.css('#profile-link'));
+            return profileLink.click();
+        }).then(function() {
+            // Get profile card modal element using className profile-popup
+            const modalContent = element(by.css('.profile-popup'));
+            chaisePage.waitForElement(modalContent);
+            expect(modalContent.isDisplayed()).toBeTruthy();
+            done();
         });
     });
 
     it('should close the profile card on click of X on the modal window', function(done) {
-         var closeLink = element(by.css('.modal-close'));
-         var modalContent = element(by.css('.modal-content'));
-         browser.wait(EC.elementToBeClickable(closeLink), browser.params.defaultTimeout);
-         chaisePage.clickButton(closeLink).then(function(){
+        const closeLink = element(by.css('.modal-close'));
+        const modalContent = element(by.css('.profile-popup'));
 
-            browser.wait(EC.not(EC.presenceOf(modalContent)), browser.params.defaultTimeout);
+         chaisePage.clickButton(closeLink).then(function(){
+            chaisePage.waitForElementInverse(modalContent, 2000);
```

Comment:
Instead of `2000` use `browser.params.defaultTimeout`. That's a value we set as part of the setting up the test environment for the wait statements.